jetty-util/src/main/java/org/eclipse/jetty/util/FuturePromise.java

```diff
@@ -118,6 +118,27 @@ public C get() throws InterruptedException, ExecutionException
             throw (CancellationException)new CancellationException().initCause(_cause);
         throw new ExecutionException(_cause);
     }
+    
+    /**
+     * Return the result if completed successfully
+     * or rethrow the Exception
```

Comment:
Need to be clearer here that the `Exception` thrown is not wrapped with `ExecutionException` like it is with `get()`.

Reply:
Done.